### ISSUE TITLE
Capture with cursor and mouse hover

### ISSUE THREAD

**Issue by nollinvoyd** [greenshot/greenshot#501]
I pretty new to Greenshot.  

I use to use Lightshot.  Whether or nor I wished to capture the muose pointer, I was able to place the cursor in my preferred location in the image, then select Prnt scrn, after which I selected the region.

My current task is creating images of my desktop Start Menu, mouse hovering over individual apps, one at a time, to prompt their titles so they will be captured in the image.

Currently, I have been unsuccessful capturing the mouse at all, regardless of where I want it to appear.

**Comment by FF-Brown** [comment]
Hello! Normally Greenshot captures the mousepointer by default (I usually turn that off), so I'm surprised you're having this issue. You can double-check that you have this preference enabled by right-clicking the Greenshot icon in the system tray and hovering over Quick preferences. If the 'Capture mousepointer' option is enabled, there should be a checkmark next to it as shown in my screenshot. (I also included the mousepointer in the screenshot just to show that it's possible)
![image](https://github.com/greenshot/greenshot/assets/54860453/e246440a-bf29-4036-b3ed-39f993aad00e)

Could you check what version of Greenshot you're using? ('About Greenshot' in that same menu)

If you're not able to get that working, you could find a png of a mousepointer with a transparent background and import it into your screenshots. There are several ways to do this. I find it easiest to have a screenshot open in the editor, find the picture I want to import in File Explorer, and Open With -> Greenshot, then hover over 'Open in image editor' until the option appears to open in my existing editor.

**Comment by BatmanAoD** [comment]
I'm on a Mac, where the preferences page looks different, and I don't see an option for capturing the mouse pointer. Additionally, it seems not to be captured by default.

<img width="469" alt="Image" src="https://github.com/user-attachments/assets/53e94f53-9d52-46c3-a7db-804acc58babc" />

**Comment by CodaCodr** [comment]
Same issue here, Windows 10. No mousepointer capture.

Greenshot 1.2.10 build 6, 64bit.

**Comment by FEA-eng** [comment]
> Same issue here, Windows 10. No mousepointer capture.
> 
> Greenshot 1.2.10 build 6, 64bit.

Indeed, mouse capture seems to be broken, at least on Windows 10 and with Greenshot 1.3.300.

**Comment by Lakritzator** [comment]
> I'm on a Mac, where the preferences page looks different, and I don't see an option for capturing the mouse pointer. Additionally, it seems not to be captured by default.

different story, this repository is about Greenshot for Windows.

**Comment by zachpoblete** [comment]
I'm on Windows 11, and even though I have **Capture mousepointer** enabled, my cursor is not captured when I take screenshots.
